styles.css

```diff
@@ -0,0 +1,3 @@
+header {
+    background-color: rgb(170, 170, 146);
+}
```

Comment:
- [OPTIONAL] According to the project requirements 👇🏻, the style.css file should include one class with styles for your header `h1` (make in your favorite color). Kindly add this 

![15](https://user-images.githubusercontent.com/79058364/184992901-0f5bb6a2-14bf-46c2-863b-bef56fe020bc.png)